docs/reference/contrib/modeladmin/indexview.rst

```diff
@@ -535,7 +535,7 @@ kind of interactivity using javascript:
 .. _modeladmin_thumbnailmixin:
 
 ----------------------------------------------------
-``wagtal.contrib.modeladmin.options.ThumbnailMixin``
+``wagtal.contrib.modeladmin.mixins.ThumbnailMixin``
 ----------------------------------------------------
```

Comment:
...or even better, we should fix the typo in `wagtal` to make it align again :-)